Makefile

```diff
@@ -47,7 +47,7 @@ download-SFO-%:
 
 download-COAK-%:
 	mkdir -p downloads/raw
-	$(WGET) http://nf4.netfile.com/pub2/excel/COAKBrowsable/efile_COAK_$(subst download-COAK-,,$@).zip -O \
+	$(WGET) http://nf4.netfile.com/pub2/excel/COAKBrowsable/efile_newest_COAK_$(subst download-COAK-,,$@).zip -O \
```

Comment:
So the problem is that even the one without "_newest_" also only contains the newest rows? Makes sense.

Reply:
I don't understand this comment.  The one without "newest" contains all the rows not just the newest rows that is why we are switching.